recipe/build.sh

```diff
@@ -1,49 +1,15 @@
 #!/bin/bash
 
-set -e # Abort on error.
-
-export PING_SLEEP=30s
-export WORKDIR="$( cd "$( dirname "${BASH_SOURCE[0]}" )" && pwd )"
-export BUILD_OUTPUT=$WORKDIR/build.out
-
-touch $BUILD_OUTPUT
-
-dump_output() {
-   echo Tailing the last 500 lines of output:
-   tail -500 $BUILD_OUTPUT
-}
-error_handler() {
-  echo ERROR: An error was encountered with the build.
-  dump_output
-  exit 1
-}
-
-# If an error occurs, run our error handler to output a tail of the build.
-trap 'error_handler' ERR
-
-# Set up a repeating loop to send some output to Travis.
-bash -c "while true; do echo \$(date) - building ...; sleep $PING_SLEEP; done" &
-PING_LOOP_PID=$!
-
-## START BUILD
-
 if [ $(uname) == Darwin ]; then
-    export CC="clang"
-    export CXX="clang++"
-    export FC="gfortran"
-    export MACOSX_DEPLOYMENT_TARGET="10.9"
-    export CXXFLAGS="-stdlib=libc++ $CXXFLAGS"
-    export CXXFLAGS="$CXXFLAGS -stdlib=libc++"
-    export DYLD_FALLBACK_LIBRARY_PATH="$PREFIX/lib"
```

Comment:
Many of these flags are pre-defined now.
Maybe we need to adjust some of those but if we are lucky the OS X build may just work with the defaults.